package.json

```diff
@@ -89,6 +89,11 @@
     ],
     "coveragePathIgnorePatterns": [
       "<rootDir>/build/",
+      "<rootDir>/helpers/",
```

Comment:
These shouldn't be ignored

Reply:
@amclin all of them are tested with the full build test, but it doesn't seem to be considered covered. it acts as if is not covered even though it is, like the `npm run test` is run in two different windows almost.

Reply:
For the root project, I'd rather bump the coverage thresholds. Notice some helper files have tests to ensure coverage, and not relying on the integration test.

See #28